### ISSUE TITLE
No keyboard trap accessibility improvements

### ISSUE THREAD

**Issue by trebuan202** [gchq/stroom#5401]
2.1.2 No Keyboard Trap (Partially Supports)
When you navigate to the hamburger menu ( 3 dots ) which is located in the top-right of the main screen, you are unable to navigate forward. I would expect that navigation either take you back to the left panel items or it would take you to the browsers address bar. Either way, there should be no keyboard trap here.
